roles/libvirt_manager/README.md

```diff
@@ -61,14 +61,13 @@ Specific `type_name`: `^crc.*` and `^ocp.*` are enabling specific paths in the m
 #### Example
 
 ```YAML
-cifmw_libvirt_manager_networks:
+_networks:
```

Comment:
Isn't documenting and  exposing a "private" variable a code smell? I'd say so. If this variable is encouraged to not be used I wouldn't document it, neither pass it. This case would collide with that mechanism we planed to put in place to detect avoid passing private vars into our playbooks.

Reply:
temporary for now. It's used only in the `cifmw_libvirt_manager_config.networks`. I'm waiting to see all the pieces together (ci_nmstate and others) and will converge things properly. More cleaning is also needed anyway.

Reply:
followup is on its way to remove that parameter and consume the networking definition instead.